rules/REQUEST-910-IP-REPUTATION.conf

```diff
@@ -89,23 +89,23 @@ SecRule TX:HIGH_RISK_COUNTRY_CODES "!@rx ^$" \
 # This rule checks the client IP address against a list of recent IPs captured
 # from the SpiderLabs web honeypot systems (last 48 hours).
 #
-#SecRule TX:REAL_IP "@ipMatchFromFile ip_blacklist.data" \
```

Comment:
@fgsch here

Reply:
Oops!

Yes, looks like I was overly zealous when going through it.

Reply:
Sorry. Should have caught this too. Sorry.

I assumed @fzipi had checked the correct working and I only looked through the files without actually loading them.

Negligence like this is biting us time and time again. We need travis to launch server and load the rules during the testing.

Personally, I do not have time for a PR to fix the issue today.

Reply:
Mistake aside, in all fairness I don't think we should include commented out rules.
AFAICT this is the only occurrence.

Reply:
https://github.com/SpiderLabs/owasp-modsecurity-crs/pull/973

Reply:
@dune73 We need to launch a travis test for this one, agreed. I've been ocassionally using ```modsec-rules-check``` from ModSecurity v3 for this also.